.github/workflows/ci.yml

```
@@ -15,42 +15,42 @@ jobs:
   unitTest:
     runs-on: ubuntu-latest
     steps:
-      - name: Get yarn cache dir
-        id: yarnCache
-        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Install Node.js
-        uses: actions/setup-node@v4
+      - name: Install bun
+        uses: oven-sh/setup-bun@v1
```

```suggestion
        uses: oven-sh/setup-bun@v2
```

use the action v2, released today